repos/fdbt-netex-output/src/netex-convertor/netexGenerator.ts

```diff
@@ -331,9 +330,11 @@ const netexGenerator = async (ticket: Ticket, operatorData: Operator[]): Promise
         tariff.id = coreData.lineIdName
             ? `Tariff@${coreData.ticketType}@${coreData.lineIdName}`
             : `op:Tariff@${coreData.placeholderGroupOfProductsName}`;
-        tariff.Name.$t = `${coreData.operatorName} - ${ticketIdentifier} - Fares for ${startCase(
-            coreData.ticketType,
-        )} ticket`;
+        tariff.Name.$t = `Tariff for ${
+            isFlatFareType(ticket) || isPointToPointTicket(ticket)
+                ? coreData.productNameForPlainText
```

Comment:
Whats the reason for us only adding the passengerType if it's not Flat Fare or a Point to Point ticket?

Reply:
[CFDS-37](https://deliverybackbone.ema.kpmg.com/pdstudio86/browse/CFDS-37)
Based on this ticket the logic changed for preassignedFareProduct

Reply:
[a365ca0](https://github.com/fares-data-build-tool/create-data/pull/559/commits/a365ca0abcaffa509486b4bba84c026a6833122c)